lib/gtfsplus/components/GtfsPlusVersionSummary.js

```diff
@@ -134,113 +134,117 @@ export default class GtfsPlusVersionSummary extends Component<Props, State> {
 
     const derivedVersions = versionSummaries.filter(v => v.originNamespace === version.namespace)
     return (
-      <Panel header={header}>
-        <Row>
-          <Col xs={6}>
-            <Alert>
-              {this.feedStatus()}
-              {derivedVersions.length > 0
-                ? <div>
+      <Panel>
+        <Panel.Heading>{header}</Panel.Heading>
```

Comment:
Ok, looking at https://react-bootstrap-v3.netlify.app/components/panel/#panels-heading, it looks like what we need to do for all the panels that previously had `<h3>`, preferably in this PR, is something like:
```jsx
const header = (
  <Panel.Title componentClass="h3">
    <Glyphicon glyph='check' /> GTFS+ for this Version
  </Panel.Title>
)
```
or more generally:
```jsx
<Panel>
  <Panel.Heading>
    <Panel.Title componentClass="h3">
      {title}
    </Panel.Title>
  </Panel.Heading>
  ...
</Panel>
```

Reply:
Have wrapped all titles in a05236356b3c99e603faa914c8b9eadf4d62a542.

Not sure how I missed that when I was first going through the docs! Thanks for catching it though, this looks much better